heka/meta/heka.yml

```diff
@@ -336,7 +336,8 @@ aggregator:
           threshold: 50
           function: percent
     - status: okay
-    # A policy which is typically used for stateless clusters
+    # A policy that is used to derive a cluster status based
+    # on the status okay or down status for its members.
```

Comment:
the comment should make it very clear that any other status than OKAY or DOWN will trigger an UNKNOWN status.
Also I know of at least one [GSE](https://github.com/tcpcloud/salt-formula-galera/blob/master/galera/meta/heka.yml#L70) using that policy that can receive CRITICAL AFD status.